src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,146 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    public class Node<E> {
+        private E item;
+        private Node<E> next;
+        private Node<E> previous;
+
+        Node(Node<E> previous, E element, Node<E> next) {
```

Comment:
what about access modifiers?

Reply:
There should be a private access modifier